src/coprocessor/codec/data_type/scalar.rs

```diff
@@ -73,7 +73,7 @@ macro_rules! impl_as_ref {
                     other => panic!(
                         "Cannot cast {} scalar value into {}",
                         other.eval_type(),
-                        stringify!($tt),
```

Comment:
Hmm, why did it compile?

Reply:
@BusyJay it stringified to the string `"$ tt"` 🤣